Controllers/UsersController.cs

```diff
@@ -93,10 +100,24 @@ public async Task<IActionResult> Signup(RegisterDTO model)
                 Email = user.Email,
                 roleId = model.RoleId
             });
+            return Ok();
+        }
 
-            return Created("", new { token });
+        // [Authorize(Roles = "Admin")]
+        [HttpPost("status/edit")]
+        public async Task<IActionResult> UserStatus(StatusDTO model) {
+            if (!IsValidApiRequest())
+            {
+                return ApiBadRequest("Invalid Headers!");
+            }
+            var user = Context.Employees.FirstOrDefault(z => z.Id == model.Id);
+            user.Status = model.Status;
+            Context.Employees.Update(user);
+            await Context.SaveChangesAsync();
```

Comment:
Maybe it is better to move this logic to `AppUserStore` and call it via `UpdateAsync` method?